server/channels/app/integration_action.go

```diff
@@ -497,7 +497,10 @@ func (a *App) SubmitInteractiveDialog(c request.CTX, request model.SubmitDialogR
 	defer resp.Body.Close()
 
 	var response model.SubmitDialogResponse
-	json.NewDecoder(resp.Body).Decode(&response) // Don't fail, an empty response is acceptable
+	err = json.NewDecoder(resp.Body).Decode(&response) // Don't fail, an empty response is acceptable
```

Comment:
That is not quite correct. An empty response body is actually a valid response. (See the code comment) Hence, some tests are failing. Can you please update the code to take that into account? Thanks!